src/lib/security/encryption-service.ts

```
@@ -0,0 +1,514 @@
+  /**
+   * Hash password with salt
+   */
+  async hashPassword(password: string, salt?: string): Promise<{
+    hash: string;
+    salt: string;
+  }> {
+    const finalSalt = salt || this.generateSalt();
+    
+    if (typeof window === 'undefined') {
+      // Server-side hashing
+      const crypto = require('crypto');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)